RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -366,6 +366,16 @@ class PumpOpsSynchronous {
                             events.insert(TimestampedHistoryEvent(pumpEvent: event, date: date), atIndex: 0)
                         }
                     }
+                    
+                    if let alarm = event as? PumpAlarmPumpEvent {
+                        switch alarm.alarmType {
+                        case .BatteryDepleted, .BatteryOutLimitExceeded, .DeviceReset:
+                            print("Found clock loss in pump history.  Ending history fetch.")
```

Comment:
But what I'm wondering about is this situation: there are two valid time change events *after* the battery out event: one to recover from the battery loss, and one that is caused by traveling, or daylight savings, or something else valid.